src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -146,5 +147,20 @@ public int MaxModelValidationErrors
         /// <see cref="ResponseCacheFilter"/>.
         /// </summary>
         public IDictionary<string, CacheProfile> CacheProfiles { get; }
+
+        /// <summary>
+        /// Gets a list of <see cref="IMetadataDetailsProvider"/> instances that will be used to 
+        /// create <see cref="ModelMetadata"/> instances.
+        /// </summary>
+        /// <remarks>
+        /// A provider should implement one or more of the following interfaces, depending on what
+        /// kind of details are provided:
+        /// <ul>
+        /// <li><see cref="IModelMetadataBindingDetailsProvider"/></li>
+        /// <li><see cref="IDisplayMetadataProvider"/></li>
+        /// <li><see cref="IValiationMetadataProvider"/></li>
```

Comment:
typo: `IValidationMetadataProvider`; missing "d" also occurs for the containing file and some fields of this type